keras_cv/models/utils.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022 The KerasCV Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Utility functions for models"""
+
+from tensorflow import keras
+from tensorflow.keras import layers
+
+
+def parse_model_inputs(input_shape, input_tensor):
```

Comment:
last thing we could do here is add test coverage over parse_model_inputs in `utils_test.py`

test:
- it constructs a keras.Input for input_shape
- it returns the tensor of the input_tensor is a keras tensor

Then LGTM, sorry for the late batch of comments.  Just thought of this after seeing the helper.

Reply:
Added a test case using assertIs to make sure the input tensor is returned when possible, and otherwise the shape matches up correctly